mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -31,21 +53,77 @@ AUI.add(
 									var listItem = currentTarget.ancestor('li');
 
 									listItem.remove();
+
+									if (finishedTask) {
+
+										var finishedHtml = A.Lang.sub(
+											TPL_FINISHED_TASK,
+											{
+												taskFinished: Liferay.Language.get('task-finished')
+											}
```

Comment:
Do not use ```Liferay.Language.get()```, as previously requested. It makes ajax call for a label, which is an unnecessary waste of resources. Instead, prepare localization in jsp. For example see, https://github.com/liferay/liferay-plugins-ee/blob/ee-6.2.x/portlets/lcs-portlet/docroot/connection.jspf